app/views/layouts/user_mailer.html.slim

```diff
@@ -0,0 +1,95 @@
+                        tr
+                          th.small-12.large-12.columns.first.last
+                            table
+                              tr
+                                th
+                                  table.spacer
+                                    tbody
+                                      tr
+                                        td height="35px" style="font-size:35px;line-height:35px;"
+                                          | &#xA0;
+                                  h4
+                                    b
+                                      = message.subject
+                                  hr
+                                  == yield
+                                  hr
+                                  p
+                                    i Please do not reply to this message.
+                                  table.spacer
+                                    tbody
+                                      tr
+                                        td height="35px" style="font-size:35px;line-height:35px;"
+                                        | &#xA0;
+                                th.expander
+                    table.wrapper.footer align="center"
+                      tr
+                        td.wrapper-inner
+                          table.row
+                            tbody
+                              tr
+                                th.small-12.large-12.columns.first.last
+                                  table
+                                    tr
+                                      th
+                                        p
+                                          'For more help, please contact the #{APP_NAME} Customer Contact Center via web form at
+                                          = link_to nil, Figaro.env.support_url
+                                      th.expander
+                    table.wrapper.legal align="center"
+                      tr
+                        td.wrapper-inner
+                          table.row
+                            tbody
+                              tr
+                                th.small-12.large-12.columns.first.last
+                                  table
+                                    tr
+                                      th
+                                        p.text-center
+                                          a href="#" About #{APP_NAME}
+                                          ' |
+                                          a href="#" Privacy policy
+                                      th.expander
+    /! prevent Gmail on iOS font size manipulation
+    div style="display:none; white-space:nowrap; font:15px courier; line-height:0;"
+      |
+        &#xA0; &#xA0; &#xA0; &#xA0; &#xA0; &#xA0; &#xA0; &#xA0; &#xA0; &#xA0; &#xA0; &#xA0; &#xA0; &#xA0; &#xA0; &#xA0; &#xA0; &#xA0; &#xA0; &#xA0; &#xA0; &#xA0; &#xA0; &#xA0; &#xA0; &#xA0; &#xA0; &#xA0; &#xA0; &#xA0;
```

Comment:
@sbc100 These spaces are necessary in order to prevent gmail on iOS from automatically enlarging all of the text and messing up the layout.  I bumped them up onto the same line as the `|` but for some reason the last line is automatically stripped.  My local copy has a line return.  Looks like the main application layout is the same way: https://github.com/18F/identity-idp/blob/master/app/views/layouts/application.html.slim